pkg/proxy/iptables/proxier.go

```diff
@@ -135,6 +135,9 @@ func (lkct LinuxKernelCompatTester) IsCompatible() error {
 const sysctlRouteLocalnet = "net/ipv4/conf/all/route_localnet"
 const sysctlBridgeCallIPTables = "net/bridge/bridge-nf-call-iptables"
 
+// stickyMaxAgeSeconds default value is 180 minutes
+const defaultStickyMaxAgeSeconds int = 180 * 60
```

Comment:
I don't think we should have a default here.  document and default it in the API code.